src/chart_types/xy_chart/renderer/canvas/xy_chart.tsx

```diff
@@ -172,8 +182,69 @@ class XYChartComponent extends React.Component<XYChartProps> {
     const chartSeriesTypes =
       seriesTypes.size > 1 ? `Mixed chart: ${[...seriesTypes].join(' and ')} chart` : `${[...seriesTypes]} chart`;
     const chartIdDescription = `${chartId}--description`;
+    const chartIdLabel = ariaLabel ? `${chartId}--label` : undefined;
+    const idForChartSeriesTypes = `${chartId}--series-types`;
+
+    const ariaProps: AriaProps = {};
+
+    if (ariaLabelledBy || ariaLabel) {
+      ariaProps['aria-labelledby'] = ariaLabelledBy ?? chartIdLabel;
+    }
+    if (accessibilityDescription || useDefaultSummary) {
+      ariaProps['aria-describedby'] = `${accessibilityDescription ? chartIdDescription : undefined} ${
+        useDefaultSummary ? idForChartSeriesTypes : undefined
+      }`;
+    }
+
+    const ChartLabel = (heading: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'p', id: string, label?: string) => {
+      if (!label) return null;
+      switch (heading) {
+        case 'h1':
+          return (
+            <h1 id={id} aria-label={label}>
+              {label}
+            </h1>
```

Comment:
`aria-label={label}` and the inner text of `{label}` will always match so we can get rid of the `aria-label`. Sames goes for all of the cases.

```suggestion
            <h1 id={id}>
              {label}
            </h1>
```

Reply:
That makes sense thank you b81e6f8